The stubborn_buddies package has libraries that are exported and others
that are installed without being exported.  The CMakeLists.txt file
contains an explicitly call to ament_environment_hooks to ensure
that library_path.dsv is created.  However, this is redundant and
results in multiple commands to generate the file.  This produces
the following error during the configure task:

CMake Error: Files to be generated by multiple different commands: \
  "TOPDIR/tmp-glibc/work/core2-64-oe-linux/stubborn-buddies/1.0.0-6-r0/build/ament_cmake_environment_hooks/library_path.dsv"

Signed-off-by: Rob Woolley <rob.woolley@windriver.com>

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0963f77..11c9b78 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -72,10 +72,10 @@ set(node_plugins "${node_plugins}lifecycle_talker::LifecycleTalker;$<TARGET_FILE
 
 # since the package installs libraries without exporting them
 # it needs to make sure that the library path is being exported
-if(NOT WIN32)
-  ament_environment_hooks(
-    "${ament_cmake_package_templates_ENVIRONMENT_HOOK_LIBRARY_PATH}")
-endif()
+#if(NOT WIN32)
+#  ament_environment_hooks(
+#    "${ament_cmake_package_templates_ENVIRONMENT_HOOK_LIBRARY_PATH}")
+#endif()
 
 add_executable(linktime_composition
   src/linktime_composition.cpp)
